packages/api/src/schema/db-schema.yaml

```diff
@@ -1015,8 +1024,87 @@ components:
           example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
         catalystPipelineStrategy:
           $ref: "#/components/schemas/task/properties/params/properties/upload/properties/catalystPipelineStrategy"
+    new-signing-key-payload:
+      additionalProperties: false
+      properties:
+        name:
+          type: string
+          description: Name of the signing key
+    signing-key-response-payload:
+      type: object
+      required:
+        - publicKey
+        - privateKey
+      properties:
+        $ref: "#/components/schemas/signing-key/properties"
+        privateKey:
+          type: string
+    stream-set-active-payload:
+      type: object
+      required:
+        - active
+      properties:
+        active:
+          $ref: "#/components/schemas/stream/properties/isActive"
+        hostName:
+          type: string
+          description: Hostname of the Mist server that processes that stream
+        startedAt:
+          type: number
+          description: Timestamp (in milliseconds) at which the stream started.
+          example: 1587667174725
+    webhook-patch-payload:
+      type: object
+      additionalProperties: false
+      properties:
+        name:
+          $ref: "#/components/schemas/webhook/properties/name"
+        url:
+          $ref: "#/components/schemas/webhook/properties/url"
+        events:
+          $ref: "#/components/schemas/webhook/properties/events"
+        sharedSecret:
+          $ref: "#/components/schemas/webhook/properties/sharedSecret"
+        streamId:
+          $ref: "#/components/schemas/webhook/properties/streamId"
     new-asset-from-url-payload:
```

Comment:
I believe this is used on a public API, for uploading an asset with a URL. Is that endpoint not in our `endpoints:` on `api-schema.yaml`? I believe it's `/api/asset/upload/url`

Reply:
Thanks! confirmed with Chase and added it back to `api-schema.yaml`